nautobot/core/tests/nautobot_config.py

```diff
@@ -37,32 +37,53 @@
         "HOST": os.getenv("NAUTOBOT_REDIS_HOST", "localhost"),
         "PORT": int(os.environ.get("NAUTOBOT_REDIS_PORT", 6379)),
         "DB": 2,
-        "PASSWORD": os.getenv("NAUTOBOT_DB_PASSWORD", ""),
+        "PASSWORD": os.getenv("NAUTOBOT_REDIS_PASSWORD", ""),
         "SSL": is_truthy(os.environ.get("NAUTOBOT_REDIS_SSL", False)),
         "DEFAULT_TIMEOUT": 300,
     },
     "webhooks": {
-        "HOST": "localhost",
-        "PORT": 6379,
-        "DB": 0,
-        "PASSWORD": "",
-        "SSL": False,
+        "HOST": os.getenv("NAUTOBOT_REDIS_HOST", "localhost"),
+        "PORT": int(os.environ.get("NAUTOBOT_REDIS_PORT", 6379)),
+        "DB": 2,
+        "PASSWORD": os.getenv("NAUTOBOT_REDIS_PASSWORD", ""),
+        "SSL": is_truthy(os.environ.get("NAUTOBOT_REDIS_SSL", False)),
         "DEFAULT_TIMEOUT": 300,
     },
     "check_releases": {
         "HOST": os.getenv("NAUTOBOT_REDIS_HOST", "localhost"),
         "PORT": int(os.environ.get("NAUTOBOT_REDIS_PORT", 6379)),
         "DB": 2,
-        "PASSWORD": os.getenv("NAUTOBOT_DB_PASSWORD", ""),
+        "PASSWORD": os.getenv("NAUTOBOT_REDIS_PASSWORD", ""),
         "SSL": is_truthy(os.environ.get("NAUTOBOT_REDIS_SSL", False)),
         "DEFAULT_TIMEOUT": 300,
     },
     "custom_fields": {
-        "HOST": os.getenv("REDIS_HOST", "localhost"),
-        "PORT": int(os.environ.get("REDIS_PORT", 6379)),
+        "HOST": os.getenv("NAUTOBOT_REDIS_HOST", "localhost"),
+        "PORT": int(os.environ.get("NAUTOBOT_REDIS_PORT", 6379)),
         "DB": 2,
-        "PASSWORD": os.getenv("REDIS_PASSWORD", ""),
-        "SSL": is_truthy(os.environ.get("REDIS_SSL", False)),
+        "PASSWORD": os.getenv("NAUTOBOT_REDIS_PASSWORD", ""),
+        "SSL": is_truthy(os.environ.get("NAUTOBOT_REDIS_SSL", False)),
         "DEFAULT_TIMEOUT": 900,
     },
 }
+
+redis_protocol = "rediss" if is_truthy(os.getenv("NAUTOBOT_REDIS_SSL", False)) else "redis"
+cache_ops_pwd = os.getenv("NAUTOBOT_REDIS_PASSWORD")
+cache_ops_host = os.getenv("NAUTOBOT_REDIS_HOST", "localhost")
+cache_ops_user = os.getenv("NAUTOBOT_REDIS_USER")
+cache_ops_port = int(os.getenv("NAUTOBOT_REDIS_PORT", 6379))
+
+CACHEOPS_REDIS = os.getenv(
```

Comment:
Can this look the same as `RQ_QUEUES` with a dictionary definition?

Reply:
I am waiting on #319 as it will require some rework of the caching config.